app/assets/javascripts/modules/auto_logoff.js.coffee

```diff
@@ -14,11 +14,17 @@ class app.AutoLogoff
     setInterval(( -> logoff_timer()), 1000)
 
   logoff_timer = () ->
-    if document.URL.indexOf('/login/login') > -1
+    if document.URL.indexOf('/sessions/new') > -1
```

Comment:
I think we should rather call this endpoint /session (singular)
since it's never possible to create multiple sessions from one client.